pywayland/utils.py

```diff
@@ -76,3 +78,49 @@ def close(self):
 
         os.close(self.fd)
         self.fd = None
+
+
+def wl_container_of(ptr: ffi.CData, ctype: str, member: str, ffi=ffi) -> ffi.CData:
```

Comment:
We should be able to use this in `notify_func` in `pywayland/server/listener.py`.

It might be good to make `ffi` as a keyword-only argument.